templates/pufferpanel/meta.yaml

```diff
@@ -0,0 +1,34 @@
+name: Pufferpanel
+description: https://github.com/PufferPanel/PufferPanel
+instructions: Run ./pufferpanel user create
```

Comment:
```suggestion
instructions: Run "./pufferpanel user add" in the console to create a new user.
```

Reply:
> user create lets you create a user from the CLI, user add creates a user from email with a predefined password.

Create did not work for me, only these were available.
![image](https://user-images.githubusercontent.com/43121732/226979420-d22bfc4c-bc1e-4b8b-ad20-e7f3d75716e1.png)